src/HNormalise/Torque/Parser.hs

```diff
@@ -295,12 +299,15 @@ parseTorqueDelete = do
 -- | `parseTorqueQueue` parses a complete log line denoting a queued job. Tested with Torue 6.1.x
 parseTorqueQueue :: Parser (Text, TorqueParseResult)
 parseTorqueQueue = do
-    takeTill (== ';') *> string ";Q;"   -- drop the prefix
+    string "torque: "
+    torqueDatestamp <- takeTill (== ';')
+    string ";Q;"   -- drop the prefix
```

Comment:
more copy-pasting of code, create a function for this if you need it in more than one place?

Reply:
True, here it might work :)